tests/test_material.py

```diff
@@ -1290,7 +1290,47 @@ def test_mcnp_mat0():
     assert_equal(mass, mass_exp)
 
 
+def uwuw():
```

Comment:
This test won't be found by `nosetests`. It needs to have a name like `test_*`
```suggestion
def test_uwuw():
```